dolomite-xenon/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# Docker helper
+#
+
+docker_compose_file=docker-compose.yml
+no_cache=true
+name=datashield/rock-dolomite-xenon
+tag=latest
+
+# Build Docker image
+build-image:
+	sudo docker build --no-cache=$(no_cache) -t="${name}:$(tag)" .
+
+push-image:
+	sudo docker image push ${name}:$(tag)
+
+
+up:
+	docker-compose -f $(docker_compose_file) up -d --remove-orphans
```

Comment:
Minor issue: `docker-compose` command is going to be deprecated this June, use the plugin command `docker compose`.
https://docs.docker.com/compose/migrate/